app/Http/Middleware/PrivacyInterceptionMiddleware.php

```diff
@@ -29,22 +27,25 @@ public function handle($request, Closure $next) {
             return $next($request);
         }
 
-        // Wenn die letzte Ausführung neuer ist als das Ack, redirecte mich bitte.
-        if ($user->privacy_ack_at <= $agreement->valid_at) {
+        // If the last execution is newer than the ack, please redirect me.
+        $user = auth()->user();
+        if ($user->privacy_ack_at == null || $agreement->valid_at->isAfter($user->privacy_ack_at)) {
```

Comment:
could be shortened to `if ($agreement->valid_at->isAfter($user->privacy_ack_at)) {`

Please test first though

Reply:
Nope, because privacy_ack_at is null at default.